l10nru.properties

```diff
@@ -1,15 +1,21 @@
 category=Localization
 description=Language pack for Russian
 homepageUrl=https://github.com/1c-syntax/sonar-l10n-ru
-publicVersions=7.9.1,8.0.2,8.1.1,8.2.1,8.3.1,8.4.1,8.5.1,8.6,8.7
+publicVersions=7.9.1,8.0.2,8.1.1,8.2.1,8.3.1,8.4.1,8.5.1,8.6,8.7,8.8
 archivedVersions=1.0,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,1.10,8.0,8.3,8.4,8.5,8.0.1,8.1,8.2
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-l10n-ru-plugin
 
+8.8.description=Support SonarQube 8.8
+8.8.date=2021-04-22
+8.8.sqVersions=[8.8,LATEST]
+8.8.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v8.8/sonar-l10n-ru-plugin-8.8.jar
+8.8.changelogUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/tag/v8.8
+
 8.7.description=Support SonarQube 8.7
 8.7.date=2021-02-23
-8.7.sqVersions=[8.7,LATEST]
+8.7.sqVersions=[8.7]
```

Comment:
There's an 8.7.1 version; range should include it :-/

Reply:
fix